### ISSUE TITLE
misogynistic, racist, sexist, transphobic, and ableist terms found in code

### ISSUE THREAD

**Issue by slammerhammer** [tensorflow/tensorflow#22468]
master: 2020
sanity check: 132
dumb: 8
silly: 26
crazy: 10
stupid: 5
lame: 28
sexist: 39
whitelist: 394
blacklist: 197

the amount of misogynistic, racist, sexist, transphobic, and ableist terms is unacceptable. i along with many other am disgusted at the use of these terms in the source. i would like to request that these terms be changed to something more politically correct and friendly. the code of conduct cites 

_Examples of unacceptable behavior by participants include:_

_The use of sexualized language or imagery and unwelcome sexual attention or advances
Trolling, insulting/derogatory comments, and personal or political attacks
Public or private harassment
Publishing others' private information, such as a physical or electronic address, without explicit permission
Conduct which could reasonably be considered inappropriate for the forum in which it occurs.
All TensorFlow forums and spaces are meant for professional interactions, and any behavior which could reasonably be considered inappropriate in a professional setting is unacceptable._

**Comment by a-d3v** [comment]
I agree, we should start making the world a better place and treat all people equally, even in code.

**Comment by M-BoloDyebambewe** [comment]
Agree, this code hurts me as a black man

**Comment by socqse** [comment]
Quite frankly, I am shocked by the list of horrendous words you have found. I would never have thought such vileness could be spewed in such a great project.
I am deeply saddened.

**Comment by M-BoloDyebambewe** [comment]
Sooooo, are they going to respond? Or ignore it like pretty much is expected of white people?

**Comment by slammerhammer** [comment]
> Sooooo, are they going to respond? Or ignore it like pretty much is expected of white people?

You're correct M'Bolo white people can be very ignorant at times

**Comment by Sija** [comment]
Sockpuppets at work, please ban and move along.

**Comment by ahitt6345** [comment]
I can understand why you want the other words to be removed-professionalism and all that-, but why whitelist, blacklist, and sanity check? Are they being used in a racist context? Whitelisting and blacklisting are normally used to show what has permission and what doesn't have permission and I can't see how sanity check would would be politically incorrect in any context.

**Comment by Sija** [comment]
@ahitt6345 I'd suggest you to stop wasting time on this. This issue has clearly nothing to do with stated title and/or description. It's sole purpose it's to incite emotional, name-calling atmosphere. Please, stop feeding the troll. Other usernames present in this issue here (@M-BoloDyebambewe @slammerhammer) is the same person talking to him/herself. If you're curious I'd recommend checking out all of their profiles to see how fake are they - and preferably reporting them for abuse :)
